pipeline-runner/R/gem2s-5-create_seurat.R

```diff
@@ -70,20 +74,20 @@ construct_scdata <- function(counts, doublet_score, edrops_out, sample, annot, c
 # construct metadata for each SeuratObject
 construct_metadata <- function(counts, sample, config) {
   message("Constructing metadata df...")
-  metadata <- data.frame(row.names = colnames(counts), samples = rep(sample, ncol(counts)))
```

Comment:
comment `# NOTE: any changes here must be reflected in meta_sets` looks out of date, because `meta_sets` doesn't seem to exist anymore

Reply:
removed. also added proper docs to this function